libs/domain/auth/login/src/login.component.ts

```diff
@@ -1,193 +1,178 @@
+  protected isLoading = valueType(this.loading$);
+
+  @asyncState()
+  protected hasSuccess = valueType(this.success$);
 
   @subscribe()
-  protected auth$ = this.authTrigger$.pipe(
-    tap(() => this.loading$.next(true)),
-    switchMap((user) =>
+  protected auth = this.authService.isAuthenticated().pipe(
+    tap((success) => {
+      this.loading$.next(false);
+      if (success) {
+        this.redirect();
+      }
+    })
+  );
+
+  protected override render(): TemplateResult {
+    return html`<oryx-card>
+      <oryx-heading slot="heading" appearance="h5">
+        <h1>${this.heading ?? i18n('user.login')}</h1>
+      </oryx-heading>
+
+      ${when(
+        !this.hasSuccess,
+        () => html`
+          <oryx-notification type="error">
+            ${i18n('user.login.not-valid')}
+          </oryx-notification>
+        `
+      )}
+
+      <form @submit=${this.handleLogin} method="post">
+        <oryx-input
+          .label=${i18n('user.login.email')}
+          required
+          ?hasError="${!this.hasSuccess}"
+        >
+          <input
+            type="email"
+            name="email"
+            required
+            placeholder=${i18n('user.login.email')}
+          />
+        </oryx-input>
+
+        <oryx-password-input
+          strategy="${ifDefined(this.componentOptions?.passwordVisibility)}"
+          .label=${i18n('login.password')}
+          required
+          ?hasError="${!this.hasSuccess}"
+        >
+          <input
+            type="password"
+            name="password"
+            required
+            placeholder=${i18n('login.password')}
+          />
+        </oryx-password-input>
+
+        ${this.renderLoginOptions()}
+
+        <oryx-button size="small">
+          <button ?disabled=${this.isLoading}>${i18n('user.login')}</button>
+        </oryx-button>
+      </form>
+    </oryx-card>`;
+  }
+
+  protected renderLoginOptions(): TemplateResult | void {
+    if (
+      !this.componentOptions.enableForgotPassword &&
+      !this.componentOptions.enableRememberMe
+    ) {
+      return;
+    }
+    return html`
+      <div class="options">
+        ${when(
+          this.componentOptions.enableRememberMe,
+          () => html`<oryx-checkbox>
+            <input
+              type="checkbox"
+              name="rememberMe"
+              aria-label=${i18n('user.login.remember-me')}
+            />
+            ${i18n('user.login.remember-me')}
+          </oryx-checkbox>`
+        )}
+        ${when(
+          this.componentOptions.enableForgotPassword,
+          () => html`<oryx-link>
+            <a href="#">${i18n('user.login.forgot-password?')}</a>
+          </oryx-link>`
+        )}
+      </div>
+    `;
+  }
+
+  protected handleLogin(e: Event): void {
+    e.preventDefault();
+    this.loading$.next(true);
+
+    const { email, password, rememberMe } = Object.fromEntries(
+      new FormData(e.target as HTMLFormElement).entries()
+    );
+
+    if (email && password) {
```

Comment:
Why do we need to check presence of email and password? This should be done by browser validation, both fields are required.

And why do you need to convert their values to string below? Both fields are text fields